app/controllers/admin/questions_controller.rb

```diff
@@ -9,6 +9,9 @@ def index
 
   def show
     @question = Question.find(params[:id])
+    @topic = Topic.find_by_id(@question.topic_id)
+    @t_q = @topic.title
+    @t_q = 'None' if @topic.title.nil?
```

Comment:
- variable naming is bad
- topics are not allowed without title, so no need of this check
- no need of @topic and @t_q variable